CONTRIBUTING.md

```diff
@@ -68,8 +68,3 @@ Try to be empathic.
 
 </details>
 
-## Microsoft Contributor License Agreement
```

Comment:
@ibrahimhaddad Does LF require some similar type of agreement for contributors?

Reply:
Tracked here: https://github.com/recommenders-team/recommenders/issues/1997